### PR TITLE
Update HTML and rename DataLab to Bennett Institute

### PR DESCRIPTION
**Pull request by tomodwyer**
- Fix markup and run prettier on HTML
- Update "DataLab" to "Bennett Institute" in text

Preview: https://bennett-update.covid19-trialstracker.pages.dev/

### CODE REVIEW

**Review by ghickman — APPROVED** [review]

File: docs/about/index.html [hunk 1]
```diff
@@ -323,7 +331,7 @@ <h2>Other COVID-19 Trials Transparency Resources</h2>
           </p>
           <p>
             Comments, questions or suggestions?
-            <a href="mailto:ebmdatalab@phc.ox.ac.uk">Get in touch</a>.
+            <a href="mailto:ebmBennett Institute@phc.ox.ac.uk">Get in touch</a>.
```
Comment (madwort) [inline docs/about/index.html]: this looks wrong

**Review by madwort — CHANGES_REQUESTED** [review]
search-replace errors in email addresses

File: docs/about/index.html [hunk 2]
```diff
@@ -204,8 +210,9 @@ <h2>How can I help?</h2>
         through collaboration with committed individuals with specific
         expertise. If you have time, knowledge, and an idea to improve a
         specific aspect of the project, please get in touch at
-        <a href="mailto:ebmdatalab@phc.ox.ac.uk">ebmdatalab@phc.ox.ac.uk</a>.
-        See below for other specific opportunities to contribute.
+        <a href="mailto:ebmBennett Institute@phc.ox.ac.uk"
+          >ebmBennett Institute@phc.ox.ac.uk</a
```
Comment (madwort) [inline docs/about/index.html]: also here

File: docs/about/index.html [hunk 3]
```diff
@@ -238,7 +245,8 @@ <h3>Ways to contribute</h3>
       </ul>
       <p>
         The Github page for the project is located
-        <a href="https://github.com/ebmdatalab/covid_trials_tracker-covid"
+        <a
+          href="https://github.com/ebmBennett Institute/covid_trials_tracker-covid"
```
Comment (madwort) [inline docs/about/index.html]: also here

**Review by madwort — APPROVED** [review]

**Pull request merged**